static/js/entry/zendesk_widget.js

```diff
@@ -28,3 +28,71 @@ window.zEmbed || function (e, t) {
     o.close();
 }("https://assets.zendesk.com/embeddable_framework/main.js", "odl.zendesk.com");
  /*]]>*/
+
+
+// This will execute when Zendesk's Javascript is finished executing, and the
+// Web Widget API is available to be used. The HTML for the Zendesk widget
+// may *not* have been inserted into the DOM yet.
+zE(function() {
+  // trigger onZendeskIFrameExists at the appropriate time
+  let tries = 0;
+  let intervalID = setInterval(() => {
+    tries += 1;
+    let iframe = document.querySelector("iframe.zEWidget-launcher");
+    if (iframe) {
+      clearInterval(intervalID);
+      onZendeskIFrameExists();
+    } else if (tries > 100) { // max 100 tries (10 seconds)
+      console.error("couldn't find Zendesk iframe");  // eslint-disable-line no-console
+      clearInterval(intervalID);
+    }
+  }, 100); // check every 100 milliseconds
+});
+
+// This will execute when Zendesk's <iframe> element has been inserted into
+// the DOM. The <iframe> may *not* have finished loading its content yet.
+let onZendeskIFrameExists = () => {
+  // trigger onZendeskIFrameLoaded at the appropriate time
+  let tries = 0;
+  let intervalID = setInterval(() => {
+    tries += 1;
+    let iframe = document.querySelector("iframe.zEWidget-launcher");
+    let btn = iframe.contentDocument.querySelector(".Button--launcher");
+    if (btn) {
+      clearInterval(intervalID);
+      onZendeskIFrameLoaded();
+    } else if (tries > 100) { // max 100 tries (10 seconds)
+      console.error("couldn't load Zendesk iframe");  // eslint-disable-line no-console
+      clearInterval(intervalID);
+    }
+  }, 100); // check every 100 milliseconds
+};
+
+// This will execute when Zendesk's <iframe> element has finished loading
+// its content. It would be nice if we could just use iframe.onload,
+// but because the page is loaded from a different domain, the browser
+// won't allow us to detect that event.
+let onZendeskIFrameLoaded = () => {
+  let iframe = document.querySelector("iframe.zEWidget-launcher");
+  let btn = iframe.contentDocument.querySelector(".Button--launcher");
+
+  let regularBackgroundColor = "rgba(0, 0, 0, .14)";
+  let hoverBackgroundColor = window.getComputedStyle(btn).backgroundColor;
+  // We need to set a new background color, and unfortunately,
+  // the existing background color is set with "!important".
+  // As a result, the only way to overriding this existing color is to
+  // *also* use "!important".
+  let setHover = () => {
+    btn.style.setProperty(
+      "background-color", hoverBackgroundColor, "important"
+    );
+  };
+  let unsetHover = () => {
+    btn.style.setProperty(
```

Comment:
nit: you can direct write as
```js
let unsetHover = () => {
    btn.style.setProperty(
      "background-color", 
      "rgba(0, 0, 0, .14)", 
     "important"
    );
  };
```
we dont need variable because we r not reusing these values